examples/graphics/batching-dynamic.html

```diff
@@ -55,19 +55,22 @@
         var entities = [];
         var i;
         for (i = 0; i < numInstances; i++) {
+
+            // random shape
             var shapeName = shapes[Math.floor(Math.random() * shapes.length)];
 
             var entity = new pc.Entity();
-            entity.addComponent("model", {
-                type: shapeName
-            });
 
-            // randomly assign a material
-            entity.model.material = Math.random() < 0.5 ? material1 : material2;
+            // randomly create render or model component
+            var componentType = Math.random() < 0.5 ? "render" : "model";
```

Comment:
I know it makes a nice functional test to do this here, but the primary purpose of the examples is to show the correct way to use the API. I don't think anyone is going to do what's here in their own applications. I would stick to using render components instead of model components. And we should maybe have a 'deprecated' examples folder that contains usage of legacy APIs that is not included in the examples browser sidebar.

Reply:
makes sense.